settings.adoc

```diff
@@ -23,7 +23,7 @@ ifdef::z9[]
 :web-client: Classic Web App
 :modern-client: Modern Web App
 :product-version: 9.0.0
-:product-release-date: March, 2020
+:product-release-date: Nov, 2021
```

Comment:
Why do we need to change the product's release date? This is date when Modern Web App was released to our customers. Not sure if you got confused with "Last Updated" date

Reply:
Fixed.